Streetcode/Streetcode.BLL/MediatR/Streetcode/Streetcode/Create/CreateStreetcodeHandler.cs

```diff
@@ -151,9 +157,22 @@ await _repositoryWrapper.StreetcodeImageRepository.CreateRangeAsync(imagesIds.Se
 
     private async Task AddImagesDetails(IEnumerable<ImageDetailsDto>? imageDetails)
     {
+        if(imageDetails.IsNullOrEmpty())
+        {
+            throw new HttpRequestException("There is no valid imagesDetails value", null, System.Net.HttpStatusCode.BadRequest);
+        }
+
         if(imageDetails == null)
         {
-            return;
+            throw new HttpRequestException("There is no valid imagesDetails value", null, System.Net.HttpStatusCode.BadRequest);
+        }
```

Comment:
These parts seem to be interchangeable. Do you need both of them?

Reply:
you are right, it works exactly good without the second check, I will remove it